openapi/v2/core-metadata.yaml

```diff
@@ -3108,6 +3287,496 @@ paths:
+              examples:
+                500Example:
+                  $ref: '#/components/examples/500Example'
+  /provisionwatcher/all:
+    parameters:
+      - $ref: '#/components/parameters/correlatedRequestHeader'
+      - $ref: '#/components/parameters/offsetParam'
+      - $ref: '#/components/parameters/limitParam'
+      - $ref: '#/components/parameters/labelsParam'
+    get:
+      summary: "Given the entire range of provision watchers sorted by last modified descending, returns a portion of that range according to the offset and limit parameters. Provision watchers may also be filtered by label."
+      responses:
+        '200':
+          description: "OK"
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MultiProvisionWatchersResponse'
+              example:
+                apiVersion: "v2"
+                requestId: "4e63c9b9-9cc4-45e4-9bf0-6d994a3860b4"
+                statusCode: 200
+                message: ""
+                provisionwatchers:
+                  - apiVersion: "v2"
+                    id: "1ff7762f-c432-4af0-9a5d-756bbc92744b"
+                    name: "virtual-watcher"
+                    created: 1600927134890
+                    modified: 1600927134890
+                    labels:
+                      - virtual
+                    identifiers:
+                      "address": "localhost"
+                      "port": "3[0-9]{2}"
+                    blockingIdentifiers:
+                      "port":
+                        - "397"
+                        - "398"
+                        - "399"
+                    profile: "device-virtual"
+                    service: "device-virtual"
+                    adminState: "UNLOCKED"
+                    autoEvents:
+                      - resource: "Bool"
+                        frequency: "10s"
+                        onChange: false
+        '400':
+          description: "Request is in an invalid state"
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                400Example:
+                  $ref: '#/components/examples/400Example'
+        '500':
+          description: "Internal Server Error"
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                500Example:
+                  $ref: '#/components/examples/500Example'
+  '/provisionwatcher/id/{id}':
```

Comment:
We have removed all `by id` queries that have corresponding `by name` queries. Remove this one. 
I notices the swagger of get by ids still exists for APIs that we are no implementing, i.e. Device by Id, etc.

Reply:
I'll remove other get by id as well.

Reply:
should we remove other `by id` queries in this PR ?

Reply:
Let's first discuss in this weeks's Core WG.